### DISCUSSION TITLE
[Feature] Support panoramas with `Projection Type: cylindrical`

### DISCUSSION THREAD

**Discussion by raldone01** [immich-app/immich#10068] [Feature Request]
### The bug

Images with the exif projection type set to `Projection Type: cylindrical` don't display as 360 panorama photos.

If manually set the projection type to `exiftool -ProjectionType=equirectangular $FILE` immich _seems_ to display the file correctly.

(Samsung S22 panorama mode sets the property to cylindrical`.)

### The OS that Immich Server is running on

Arch docker

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

NaN

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. Upload a file with ProjectionType cylindrical
2. View on the web version
```

**Comment by bo0tzz** [comment]
Immich only supports equirectangular at the moment, so I've converted this to a feature request.
